decentraland/unity-explorer
# Fix: disable own nametag on first person view

## What does this PR change?

Fix #733 disables own nametag when in first person view
...

## How to test the changes?



1. Launch the explorer
2. Switch between first and third person
3. Check that in first person your own nametag is not visible

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
